TBD-dragon-game/Global.gd

```diff
@@ -0,0 +1,51 @@
+extends Node
```

Comment:
moved scene switching here using autoloader, this is way cleaner
see: https://docs.godotengine.org/en/stable/tutorials/scripting/singletons_autoload.html

Reply:
Awesome